.github/workflows/ci.yml

```diff
@@ -169,6 +169,14 @@ jobs:
       if: ${{ matrix.db != 'sqlite' }}
     - name: pnpm install
       run: pnpm install --offline 
+    - name: Linting for test suite sql-mapper
+      run: cd packages/sql-mapper && npx standard; cd ../..
```

Comment:
Can you add `lint` script to the package.json and run pnpm instead of npx.
```suggestion
      run: cd packages/sql-mapper && pnpm run lint; cd ../..
```

Reply:
Ok, thank you.

Reply:
I try to add `lint` in `package.json`, with `sql-mapper` package has worked, but with other packages I get this error:
```
npm ERR! code EUNSUPPORTEDPROTOCOL
npm ERR! Unsupported URL Type "workspace:": workspace:*

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ricfio/.npm/_logs/2022-10-02T13_00_31_102Z-debug-0.log
```

I think is due to this `devDependencies` with "workspace:*" as version in the packages:
```
  "devDependencies": {
    "@platformatic/sql-mapper": "workspace:*",
...
```

Tips? It's my npm version to old?
```
npm -v
8.15.0
```

Reply:
Please use pnpm instead of npm.

Reply:
Could I replace `npx standard` with `pnpx standard`?
What you said me about `lint` question ?

Reply:
You can add to all package.json files that you use a new `lint` script and run this script with pnpm instead of run `standard` directly. (Example: https://github.com/fastify/fast-json-stringify/blob/f5f16238d02ba3e3c3dc3b32238508c3beec276d/package.json#L12.

Reply:
Ah ok, I understand... so I have not add `lint` package in devDependency. 👍

Reply:
`lint` is not a package, it's a command. You should add it in the `scripts` block, not in the `devDependency `.

Reply:
Yes, I understand.

Reply:
Ok, I have done. Thank you so much!

Reply:
> Please use pnpm instead of npm.

Ok, only now I understand from CONTRIBUTE.md that this dependency format `"@platformatic/sql-json-schema-mapper": "workspace:*"` is not supported by npm but only by pnpm. Thank you.